shapes/emoji/src/EmojiDrawer.ts

```
@@ -59,39 +72,54 @@ export class EmojiDrawer implements IShapeDrawer<EmojiParticle> {
     }
 
     particleDestroy(particle: EmojiParticle): void {
-        delete particle.emojiData;
+        delete particle.emojiDataKey;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)